build/source/driver/multi_driver.f90

```diff
@@ -266,7 +288,7 @@ program multi_driver
 ! nGRU-is the total number of GRUs of the simulation domain
 ! nHRU-is the total number of HRUs of the simulation domain
 ! hruCount-is a local variable for the total number of HRUs in a GRU
-call allocate_gru_struc(nGRU,nHRU,err,message); call handle_err(err,message)
+call allocate_gru_struc(nGRU,nHRU,strtHRU,err,message); call handle_err(err,message)
```

Comment:
When parallelizing the HRU level, the unique GRUs need to be located during allocation. And if we don't define mapping structures it will need some duplicate codes in mapping. But if we parallelize at the GRU level, it will be logical to separate them.